views/partials/searchFilter.njk

```diff
@@ -22,6 +22,10 @@
       type: "submit",
       attributes: { "data-qa": "filter-button" }
     }) }}
+    {{ govukButton({
+      text: "Download",
+      attributes: { "id": "download-button", "onclick": downloadAction }
+    }) }}
```

Comment:
Not sure if the search filter is the best place for the download button - but it looked wrong anywhere else.

Reply:
would be good to see a preview of the page so that I understand what you mean - if it is right justified and not in green then that would be fine